config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -99,6 +99,56 @@ periodics:
     testgrid-alert-email: kubernetes-sig-node-test-failures@googlegroups.com
     description: "Uses kubetest to run serial node-e2e tests (+Serial, -Flaky|Benchmark|Node*Feature)"
 
+- name: ci-kubernetes-node-kubelet-serial-containerd-flaky
+  cluster: k8s-infra-prow-build
+  interval: 4h
```

Comment:
this is too short period, if this is going to be constantly failing and you plan to check this from time to time consider a much longer interval to not waste resources

Reply:
What value do you think would be good for this? once a day?

Reply:
adding 24h.

Reply:
sounds fair